DAS/das/mqtt_playback_V2.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+from das.utils import logger
+
+parser = argparse.ArgumentParser(
+    description="MQTT playback",
+    add_help=True,
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+
+parser.add_argument(
+    "filepath", action="store", type=str, help="""Filepath of the csv log"""
+)
+
+parser.add_argument(
+    "--host",
+    action="store",
+    type=str,
+    default="localhost",
+    help="""Address of the MQTT broker.""",
+)
+
+parser.add_argument(
+    "-s",
+    "--speed",
+    action="store",
+    type=float,
+    default=1,
+    help="""Playback speed up. Default is 1x""",
```

Comment:
Can remove `Default is 1x` since you added the default formatter in argparse:

![Screen Shot 2020-09-05 at 4 01 37 pm](https://user-images.githubusercontent.com/18709969/92298915-15027380-ef91-11ea-8e04-3d9b96cd28f4.png)